plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/EclipseProjectCreator.java

```diff
@@ -0,0 +1,21 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IWorkspaceRoot;
+import org.eclipse.core.resources.ResourcesPlugin;
+
+/**
+* Utility to make a new Eclipse project with the App Engine Standard Nature in the workspace.  
```

Comment:
done. For now this is just a stub. It's the next piece I'm going to work on once this is in place.